addons/longhorn/template/base/template/patch-defaults-cm.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: longhorn-default-setting
+  namespace: longhorn-system
+data:
+  default-setting.yaml: |-
+    storage-over-provisioning-percentage: ${LONGHORN_STORAGE_OVER_PROVISIONING_PERCENTAGE}
+    upgrade-checker: false
+    priority-class: system-node-critical
```

Comment:
won't this override the _entire_ `default-setting.yaml` key, and not just these three items?

Reply:
yes but defaults are empty. id rather patch so as not to override/duplicate things like metadata.labels